src/components/Form/Input/UnderlineTextField.tsx

```diff
@@ -0,0 +1,52 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+export interface IUnderlineTextField extends React.ComponentProps<'input'> {
+  invalid: boolean;
+  variant: 'small' | 'large' | 'responsive';
+}
+
+export const UnderlineTextField = ({ invalid = false, variant = 'responsive', ...rest }: IUnderlineTextField) => {
+  return <StyleInput invalid={invalid} variant={variant} {...rest}></StyleInput>;
```

Comment:
별도의 children이 없을 경우에는 self-closing tag를 사용해주시면 좋을 것 같습니다~
```suggestion
  return <StyleInput invalid={invalid} variant={variant} {...rest} />;
```